### DISCUSSION TITLE
Router on Multi Domain

### DISCUSSION THREAD

**Discussion by gregbm** [Ylianst/MeshCentral#5037] [General]
Hi!
Does anyone know how to connect the router to a secend domain?
I've configured a second domain but the router only connects the the default one.
The second domain looks like www.mydomain.xy/customer. Entering this as the serverstring
in the router still connects to the default domain.

Greg

**Comment by si458** [comment]
i believe you have to use subdomains and domains, paths wont work
https://www.youtube.com/watch?v=eqInqM9olXA - video explaining multiple domains/accounts

**Comment by gregbm** [comment]
Ahhh, OK, I'll try this. Thank you!
